packages/sdk/src/lib.ts

```diff
@@ -123,6 +129,12 @@ export class PhotonClient {
         query: {
           fetchPolicy: 'cache-first',
           errorPolicy: 'all'
+        },
+        mutate: {
```

Comment:
Note to @captDaylight to look at and test this

Reply:
this worked perfectly on the create templates mutation, but not seeing it happen for deleting templates.

Reply:
I actually don't think it works in either place. We need to change the appsync cache